### PR TITLE
Use single-quote to ensure code is valid when executed on Windows

### LINKED ISSUE #2192: org.gradle.api.GradleException: Glean code generation failed
Issue seen on Fenix (probably reproducing just on Windows) - https://github.com/mozilla-mobile/fenix/issues/27053

### PR DESCRIPTION
**Pull request by badboy**
Should fix https://github.com/mozilla-mobile/fenix/issues/27053

Fixes #2192

### CODE REVIEW

File: gradle-plugin/src/main/groovy/mozilla/telemetry/glean-gradle-plugin/GleanGradlePlugin.groovy [hunk 1]
```diff
@@ -104,7 +104,7 @@ else:
         sys.exit(1)
     else:
         # We check MAJOR.MINOR only
-        expected_ver = expected_version.split(".")
+        expected_ver = expected_version.split('.')
         expected_maj, expected_min = int(expected_ver[0]), int(expected_ver[1])
         current_ver = found_version.split(".")
```
Comment (Dexterp37) [inline gradle-plugin/src/main/groovy/mozilla/telemetry/glean-gradle-plugin/GleanGradlePlugin.groovy:109]: I think we need to change this as well.

**Review by Dexterp37 — CHANGES_REQUESTED** [review]

**Comment by Dexterp37** [comment]
Fixes mozilla-mobile/fenix#27053

**Review by Dexterp37 — APPROVED** [review]
I just tested this on my Windows machine within A-C. This works for me!

**Comment by sv-ohorvath** [comment]
Do you have an estimate of when this will land on Fenix?

**Comment by Dexterp37** [comment]
> @badboy Do you have an estimate of when this will land on Fenix?

We need to cut a new release. This will likely happen next week as this week we're at a company wide meeting.

**Comment by sum20156** [comment]
@Dexterp37 when it will be done

**Comment by ioanarusiczki** [comment]
@Dexterp37 I tried yesterday to make a build for AMO stage addon installs - > debug variant, I still get this error

**Comment by Dexterp37** [comment]
> @Dexterp37 when it will be done

@sum20156 a new release was cut yesterday and it's now being picked up by the products using Glean.

> @Dexterp37 I tried yesterday to make a build for AMO stage addon installs - > debug variant, I still get this error

@ioanarusiczki what's the specific product and steps to reproduce? If it's addons then it's the Glean JS SDK probably, which lives in mozilla/glean.js

**Comment by sum20156** [comment]
@Dexterp37 to reproduce..I just freshly cloned the repo...branch is main and the build variant is debug

**Comment by Dexterp37** [comment]
> @Dexterp37 to reproduce..I just freshly cloned the repo...branch is main and the build variant is debug

@sum20156 sorry for the confusion, I was asking @ioanarusiczki about how to reproduce, as their problem is likely for another product. Fenix is going to use the freshly cut Glean version soon.

**Comment by ioanarusiczki** [comment]
@Dexterp37 It's the same as @sum20156 said, I cloned the repo (a while ago), yesterday I've updated the repo. 
I must change a few lines so the build would let the addon installation happen on AMO stage, it's here: 
https://github.com/mozilla/addons-frontend/pull/10021/commits/e65a006767ea2e9f21d4454ace9a386215874c5b

My variant is debug. 

I get the following errors:

errors in AS

**Comment by Dexterp37** [comment]
Oh, I see, so it's still Fenix (sorry I was confused by what addons-frontend was).

Then the answer is the same: the latest version is getting adopted by new products. The new fix will like be in Fenix tomorrow. 

Sorry for the delay, it's been all hands + flying back so this week stuff is mostly slow.

**Comment by ioanarusiczki** [comment]
@Dexterp37 No problem, thanks for the info.

**Comment by ioanarusiczki** [comment]
@Dexterp37 thanks! it worked 

epic win

**Pull request merged**